### PR TITLE
os/arch/arn/src/amebasmart: Modify i2c pins for board revision 5th

### PR DESCRIPTION
**Pull request by ziyik**
-Modify pins for i2c0 and i2c1 according to board revision 5th

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@abhishek-samsung Could you confirm this?

File: os/arch/arm/src/amebasmart/amebasmart_i2c.c [hunk 1]
```diff
@@ -301,8 +301,13 @@ static const struct amebasmart_i2c_config_s amebasmart_i2c0_config = {
 	//.busy_idle = CONFIG_I2C0_BUSYIDLE,
 	//.filtscl = CONFIG_I2C0_FILTSCL,
 	//.filtsda = CONFIG_I2C0_FILTSDA,
+#if CONFIG_RTL8730E_BOARD_REVISION >= 5
+	.scl_pin = PA_29,
+	.sda_pin = PA_28,
+#else
```
Comment (abhishek-samsung) [inline os/arch/arm/src/amebasmart/amebasmart_i2c.c:307]: https://github.com/Samsung/TizenRT/blob/9522f9b6f99fcf192de3cb2a6de3db240b46375f/os/arch/arm/src/amebasmart/amebasmart_i2c.c#L359-L365

I2C port 2 is already using PA_28 and PA_29 and it's the same mentioned in the data shared by h/w team. So, I am not sure if its the right change.

Comment (ziyik) [reply os/arch/arm/src/amebasmart/amebasmart_i2c.c]: This is a mistake on my side, please ignore this change, I have reverted it.

**Review by sunghan-chang — COMMENTED** [review]
i2c0 also has pin map changed.
pa_9 -> pb29
pa_10 -> pb30

Could you check it as well?

**Comment by ziyik** [comment]
> #if CONFIG_RTL8730E_BOARD_REVISION >= 5
> 	.scl_pin = PA_29,
> 	.sda_pin = PA_28,
> #else

Yes, you are right

**Review by sunghan-chang — APPROVED** [review]

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**